lib/SimpleSAML/Metadata/MetaDataStorageHandler.php

```diff
@@ -256,6 +257,38 @@ public function getPreferredEntityIdFromCIDRhint($set, $ip)
         return null;
     }
 
+    /**
+     * This function loads the metadata for entity IDs in $entityIds. It is returned as an associative array
+     * where the key is the entity id. An empty array may be returned if no matching entities were found
+     * @param array $entityIds The entity ids to load
+     * @param string $set The set we want to get metadata from.
+     * @return array An associative array with the metadata for the requested entities, if found.
+     */
+    public function getMetaDataForEntities(array $entityIds, $set)
+    {
+
```

Comment:
This newline does not belong here as per PSR-2

Reply:
Fixed